File: qc-storage/src/main/java/com/eager/questioncloud/storage/question/PostCommentJpaRepository.java
package com.eager.questioncloud.storage.question;

import java.util.Optional;
import org.springframework.data.jpa.repository.JpaRepository;

public interface PostCommentJpaRepository extends JpaRepository<PostCommentEntity, Long> {
    Optional<PostCommentEntity> findByIdAndWriterId(Long id, Long userId);
}

File: qc-infra/qc-rdb/src/main/kotlin/com/eager/questioncloud/post/repository/PostCommentJpaRepository.kt
package com.eager.questioncloud.post.repository

import com.eager.questioncloud.post.entity.PostCommentEntity
import org.springframework.data.jpa.repository.JpaRepository
import java.util.*

interface PostCommentJpaRepository : JpaRepository<PostCommentEntity, Long> {
    fun findByIdAndWriterId(id: Long, userId: Long): Optional<PostCommentEntity>
}
